### ISSUE TITLE
Mapping Saturation Indicator results should include best fitting curve description

### ISSUE THREAD

**Issue by matthiasschaub** [GIScience/ohsome-quality-analyst#175]
Once a best fitting curve is found (sigmoid with two jumps or sigmoid with three jumps etc.) the returned result of the indicator should contain which one it is.

- [x] Add a new attribute the Mapping Saturation e.g: `best_fitting_curve: str = "double sigmoid"`

**Comment by SlowMo24** [comment]
would it hurt to provide the exact formula? There is a plethora of formulas (even with a specific # of steps), fitting mechanisms and it may also depend on the starting values? So the exact formula with the fitted parameters in a second attribute like `"formula": "2x²+4x+9"` or even with sub attributes like 

```json
"fitted_function_formula": "ax²+bx+c",
"formula_parameter:a": 2,
"formula_parameter:b": 4,
"formula_parameter:c": 9
```

**Comment by matthiasschaub** [comment]
As currently implemented in PR #170 the whole model object including its attributes (name, formula, fitted_data, metric) will be returned.
